docs/wxm-devices/D1-bundle/D1-troubleshooting.mdx

```diff
@@ -65,7 +66,8 @@ especially for the variables of wind speed and wind direction. However, you shou
 e.g., from a weather station, which is located within a distance of 1km/0.6mi.
 If there are deviations, please check [Best Practices for Proper Station Installation](/wxm-devices/D1-bundle/install-weather-station#best-practices-for-proper-station-installation) for guidelines about the installation of the weather station.
 
-### WG1200 Gateway displays "Station Reported Abnormal Values".
+### Device Alert "Station Reported Abnormal Values".
```

Comment:
Add a `:` please and remove the dot at the end ----> `Device Alert: "Station Reported Abnormal Values"`

Also, move this above "Temperature looks higher than expected" so that all Device Alerts are at the start of this section together.